web/packages/teleport/src/Navigation/NavigationDropdown.tsx

```diff
@@ -33,9 +33,8 @@ import { getIcon } from 'teleport/Navigation/utils';
 
 import { useTeleport } from 'teleport';
 
-import type { Location } from 'history';
-
 import type { TeleportFeature } from 'teleport/types';
+import type * as history from 'history';
```

Comment:
```suggestion
import type * as History from 'history';
```

Reading the code it's a bit confusing seeing calls to `history` and it not being "history".